cadc-util/src/test/java/ca/nrc/cadc/net/HttpTransferTest.java

```diff
@@ -70,13 +70,17 @@
 package ca.nrc.cadc.net;
 
 import ca.nrc.cadc.auth.SSOCookieCredential;
+import ca.nrc.cadc.util.HexUtil;
 import ca.nrc.cadc.util.Log4jInit;
 import java.io.ByteArrayOutputStream;
 import java.io.File;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.net.HttpURLConnection;
 import java.net.MalformedURLException;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
 import java.util.Date;
```

Comment:
These imports can be removed... not a big deal.  I guess there's no checksytle on test classes.

Reply:
Updated the spelling and removed unused imports. No, checkstyle doesn't run on test classes, not sure why the distinction.